setuptools/command/editable_wheel.py

```diff
@@ -530,11 +530,20 @@ def __call__(self, wheel: "WheelFile", files: List[str], mapping: Dict[str, str]
 
         name = f"__editable__.{self.name}.finder"
         finder = _normalization.safe_identifier(name)
+        return finder, name, mapping, namespaces_
+
+    def get_implementation(self) -> Iterator[Tuple[str, bytes]]:
```

Comment:
Why hide the true type of `Generator[tuple[str, bytes], None, None]` here? Is there a reason for only wanting to interact with an `Iterator` ?

Reply:
My personal opinion is that `Generator[tuple[str, bytes], None, None]` is a beast/convoluted to understand...
`Iterator[Tuple[str, bytes]]` is clearer.

(also, we don't use the "coroutine" abilities of the "generator"[^1], i.e. there is no `generator.send`, so I don't is relevant to have the explicit type there).

[^1]: Python nomenclature with the `generator` vs `coroutine` is weird... Here I mean `coroutine` as in other languages, without the weird `async` that ended up messed in the Python concepts.

Reply:
Thanks! Just double checking the intention since I'll need to update the typeshed stubs for version 70